teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -447,12 +462,41 @@ private OptionalInt getP2pLowerBound() {
   }
 
   private OptionalInt getP2pUpperBound() {
-    if (p2pUpperBound.isPresent() && p2pLowerBound.isPresent()) {
-      return p2pLowerBound.getAsInt() > p2pUpperBound.getAsInt() ? p2pLowerBound : p2pUpperBound;
-    }
     return p2pUpperBound;
   }
 
+  private List<String> getStaticPeersList() {
+    final List<String> staticPeers = new ArrayList<>(p2pStaticPeers);
+
+    if (p2pStaticPeersUrl != null) {
+      try {
+        final Optional<InputStream> maybeStream =
+            ResourceLoader.urlOrFile().load(p2pStaticPeersUrl);
```

Comment:
Q: it looks like if the url is unavailable or unreachable, we won't start the node. If this is the case, we need to remember to add a note/warning in the docs.

Reply:
Fail fast is normal and healthy here i think. it's also consistent with what we do elsewhere, i can update our doc PR.